src/pages/partnerorg.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Link } from "gatsby";
+
+import getPartnerOrganization from "../hooks/get-partner-organization";
+import Layout from "../components/layout";
+import SEO from "../components/seo";
+
+const QueryPartnerOrgPage = () => {
+  const data = getPartnerOrganization();
+  return (
+    <Layout>
+      <SEO title="Home" />
+      <h1>Partner Organizations</h1>
+      <p>Data</p>
+      {data.allMarkdownRemark.edges.map(post => (
+        <div key={post.node.id}>
+          <p>{post.node.frontmatter.title}</p>
+          <p>{post.node.frontmatter.email}</p>
+          <p>{post.node.frontmatter.photo}</p>
+          <p>{post.node.frontmatter.link}</p>
+          <p>{post.node.frontmatter.description}</p>
+        </div>
+      ))}
```

Comment:
Is it possible to put this logic before returning the orgs.

So when a user calls this function, they can do data[0].title directly and not data.allMarkdownRemark.edges[0].node.frontmatter.title

Reply:
Will check that

Reply:
So the user can do something like that.

 const data = getPartnerOrganization();

<p>{data[0].title}</p>
<p>{data[0].email}</p>
<p>{data[0].photo}</p>
<p>{data[0].link}</p>
<p>{data[0].description}</p>